public/index.html

```diff
@@ -24,7 +24,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>Astronauts In Space Now</title>
```

Comment:
`<HeaderComponent astronautCount={ # of astros here} />`

Reply:
Okay, I added a header component that does this, but I did it in AstronautsTable and maybe that was not the move. Hmm perhaps I will move it out to app.js